native/acter/api.rsh

```diff
@@ -128,13 +128,13 @@ object NewsEntryDraft {
     fn add_image_slide(body: string, url: string, mimetype: string, size: Option<u32>, width: Option<u32>, height: Option<u32>, blurhash: Option<string>) -> Future<Result<bool>>;
 
     /// create news slide for audio msg
-    fn add_audio_slide(body: string, url: string, secs: Option<u32>, mimetype: Option<string>, size: Option<u32>);
+    fn add_audio_slide(body: string, url: string, mimetype: string, secs: Option<u32>, size: Option<u32>);
```

Comment:
am I missing something or how are these changes here and in the `attachments.rs` related to this upgrade?

Reply:
Fixed